### PR TITLE
README.md and RST Doc Refactoring / LICENSE.md Inclusion / File-Level Doc-Blocks

### PR DESCRIPTION
**Pull request by robfrawley**
- Updated README.md (see https://github.com/robfrawley/LiipImagineBundle/blob/feature-license/README.md to reference how the changes are rendered).
- Refactored RST docs (see https://src.run/docs/symfony-build/liipimaginebundle/index.html to reference how the changes are rendered).
- Added LICENSE.md
- Added file-level doc-blocks

### CODE REVIEW

**Comment by lsmith77** [comment]
don't have time today to review in detail but looks like a good idea in general.

however I would not add `` .editorconfig``.

**Comment by robfrawley** [comment]
Just wondering why the dislike of an `.editorconfig` file? No worries, though: I'll remove that shortly. I have another large pull request waiting in-the-wings for after this one is approved that raises test coverage to 78% and refactors some tests/code: https://github.com/liip/LiipImagineBundle/compare/master...robfrawley:feature-refactor-tests?expand=1

**Comment by robfrawley** [comment]
Rebased on top of latest master, fixes merge conflict caused by pull request #779.

**Comment by robfrawley** [comment]
See the updated RST output at https://src.run/docs/symfony-build/liipimaginebundle/index.html

**Comment by antoligy** [comment]
I'm quite curious about the relicense - Any particular reason why we should move away from MIT to CCv3 BY-NC-SA?
https://github.com/liip/LiipImagineBundle/blob/1.0/composer.json#L7

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+* **2016-09-03**: [License] Created markdown version of *Creative Commons
```
Comment (antoligy) [inline CHANGELOG.md]: LICENSE.md is the MIT license!

**Comment by antoligy** [comment]
For the header is it worth adding an [`@license MIT`](https://phpdoc.org/docs/latest/references/phpdoc/tags/license.html)?

**Comment by antoligy** [comment]
Given the status of our CHANGELOG, honestly I think it flat-out be worth removing from the scope of this PR so that we can decide what to do with it at a later date.  Sorry to be so much trouble!

**Comment by robfrawley** [comment]
@antoligy: Yeah, so I don't even need to read the link you provided to know what you're getting at and agree; this PR ballooned way out of its initial scope and has just kept getting bigger. Best kill it now. ;-)

Closed in favor of #788, #789, #790, and #791.

**Pull request closed without merging**